docs/source/overview.rst

```diff
@@ -4,17 +4,60 @@ Overview of Iroha
 What are the key features of Iroha?
 -----------------------------------
 
-Where Iroha can be used?
+- Simple deployment and maintenance
+- Variety of libraries for developers
+- Role-based access control
+- Modular design, driven by command–query separation principle
+- Assets and identity management
+
+In our quality model, we focus on and continuously improve:
+
+- Reliability (fault tolerance, recoverability)
+- Performance Efficiency (in particular time-behavior and resource utilization)
+- Usability (learnability, user error protection, appropriateness recognisability)
+
+Where can Iroha be used?
 ------------------------
 
+Hyperledger Iroha is a general purpose permissioned blockchain system that can be used to manage digital assets, identity, and serialized data.
+This can be useful for applications such as interbank settlement, central bank digital currencies, payment systems, national IDs, and logistics, among others.
+
+For a detailed description please check our `Use Case Scenarios section <http://iroha.readthedocs.io/en/latest/use_cases/>`_.
+
 How is it different from Bitcoin or Ethereum?
 ---------------------------------------------
 
+Bitcoin and Ethereum are designed to be permissionless ledgers where anyone can join and access all the data.
+They also have native cryptocurrencies that are required to interact with the systems.
+
+In Iroha, there is no native cryptocurrency. Instead, to meet the needs of enterprises, system interaction is permissioned, meaning that only people with requisite access can interact with the system. Additionally, queries are also permissioned, such that access to all the data can be controlled.
+
+
+One major difference from Ethereum, in particular, is that Hyperledger Iroha allows users to perform common functions, such as creating and transferring digital assets, by using prebuilt commands that are in the system.
+This negates the need to write cumbersome and hard to test smart contracts, enabling developers to complete simple tasks faster and with less risk.
+
 How is it different from the rest of Hyperledger frameworks or other permissioned blockchains?
 ----------------------------------------------------------------------------------------------
 
+Iroha has a novel, Byzantine fault tolerant consensus algorithm (called YAC [#f1]_) that is high-performance and allows for finality of transactions with low latency.
+Other frameworks either focus more on probabilistic consensus algorithms, such as Nakamoto Consensus, or sacrifice Byzantine fault tolerance.
+
+Also, Iroha's built-in commands are a major benefit compared to other platforms, since it is very simple to do common tasks such as create digital assets, register accounts, and transfer assets between accounts.
```

Comment:
Also you can mention that is narrows the attack vector, improving overall security of the system, as there are less things to fail